opencti-platform/opencti-graphql/src/domain/stixCyberObservable.js

```diff
@@ -389,6 +389,9 @@ export const artifactImport = async (context, user, args) => {
   const artifact = await addStixCyberObservable(context, user, artifactData);
   const meta = { version };
   await upload(context, user, `import/${artifact.entity_type}/${artifact.id}`, resolvedFile, { entity: artifact, meta });
+  if (!upload) {
+    throw FunctionalError('Something went wrong with the file upload');
+  }
```

Comment:
the upload variable seems undefined

```suggestion
  const upload = await upload(context, user, `import/${artifact.entity_type}/${artifact.id}`, resolvedFile, { entity: artifact, meta });
  if (!upload) {
    throw FunctionalError('Something went wrong with the file upload');
  }
```